api/base/middleware.py

```diff
@@ -410,3 +410,12 @@ def set_sloan_cookie(self, name: str, value, url, request, resp, custom_domain=N
         # Browsers won't allow use to use these cookie attributes unless you're sending the data over https.
         resp.cookies[name]['secure'] = True
         resp.cookies[name]['samesite'] = 'None'
+
+    def check_browseable_api(self, request):
+        """
+        Should return true if browseable api is requested.
+        :param request:
+        :return:
+        """
+        return request.environ.get('HTTP_ACCEPT') == 'text/html' or \
```

Comment:
looks like my `Accept` header in chrome is actually:
```
text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.9
```
might be worth testing that that mess works too.